File: examples/java-gradle/src/test/java/com/example/AppTest.java
package com.example;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

class AppTest {
    @Test
    void greetReturnsCorrectMessage() {
        assertEquals("Hello, world!", App.greet("world"));
    }

    @Test
    void greetReturnsCorrectMessageForAnyName() {
        assertEquals("Hello, Mustfile!", App.greet("Mustfile"));
    }
}

File: examples/kotlin-gradle/src/test/kotlin/com/example/AppTest.kt
package com.example

import kotlin.test.Test
import kotlin.test.assertEquals

class AppTest {
    @Test
    fun greetReturnsCorrectMessage() {
        assertEquals("Hello, world!", greet("world"))
    }

    @Test
    fun greetReturnsCorrectMessageForAnyName() {
        assertEquals("Hello, Mustfile!", greet("Mustfile"))
    }
}
